test/integration/crud/find.test.js

```diff
@@ -450,52 +450,6 @@ describe('Find', function () {
     }
   });
 
-  it('shouldCorrectlyPerformFindByWhere', {
-    metadata: {
-      requires: {
-        mongodb: '<=4.2.x',
```

Comment:
This still runs on 4.2 servers

Reply:
I've brought the test back now with the mongodb metadata removed.